platforms/atspi-common/src/error.rs

```diff
@@ -3,20 +3,32 @@
 // the LICENSE-APACHE file) or the MIT license (found in
 // the LICENSE-MIT file), at your option.
 
-#[derive(Debug, thiserror::Error)]
+use std::fmt;
+
+#[derive(Debug)]
 pub enum Error {
-    #[error("defunct")]
     Defunct,
-    #[error("unsupported interface")]
     UnsupportedInterface,
-    #[error("too many children")]
     TooManyChildren,
-    #[error("index out of range")]
     IndexOutOfRange,
-    #[error("too many characters")]
     TooManyCharacters,
-    #[error("unsupported text granularity")]
     UnsupportedTextGranularity,
 }
 
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let msg = match self {
```

Comment:
I'm not a big fan of unnecessarily shortening variable names, the meaning of "msg" might not be obvious to a non-English speaker. The match expression could easily be moved into the function call.